src/thirdparty/axom/CLI11.hpp

```diff
@@ -4746,7 +4746,7 @@ class App {
     Option *help_all_ptr_{nullptr};
 
     /// This is the formatter for help printing. Default provided. INHERITABLE (same pointer)
-    std::shared_ptr<FormatterBase> formatter_{new Formatter()};
+    std::shared_ptr<FormatterBase> formatter_{std::make_shared<Formatter>()};
```

Comment:
This was the workaround needed to bypass the compiler error for rocm@5.6.0.

Does this change require updating the  `CLI11.patch` file? If so, is there a preferred command/tool arguments to generate the updated patch file?

Reply:
I just use `git diff &> some_name.patch` when you have only those changes uncommited to git.